dataplane/linux/vxlan_mgr.go

```diff
@@ -137,6 +188,20 @@ func (m *vxlanManager) getLocalVTEP() *proto.VXLANTunnelEndpointUpdate {
 	return m.myVTEP
 }
 
+func (m *vxlanManager) getLocalVTEPParent() (netlink.Link, error) {
+	return m.getParentInterface(m.getLocalVTEP())
+}
+
+func (m *vxlanManager) GetRouteTables() []routeTable {
+	rts := []routeTable{m.routeTable}
+
+	if m.parentRouteTable != nil {
```

Comment:
I suggest adding a comment here explaining why we have this nil check - it's somewhat abnormal so worth explaining for future readers.

Reply:
OK will do.